libcxx/include/__memory/uses_allocator_construction.h

```diff
@@ -36,7 +37,10 @@ inline constexpr bool __is_std_pair = false;
 template <class _Type1, class _Type2>
 inline constexpr bool __is_std_pair<pair<_Type1, _Type2>> = true;
 
-template <class _Type, class _Alloc, class... _Args, __enable_if_t<!__is_std_pair<_Type>, int> = 0>
+template <class _Tp>
+inline constexpr bool __is_cv_std_pair = __is_std_pair<remove_cv_t<_Tp>>;
```

Comment:
This seems to be the resolution of http://wg21.link/LWG3677. We should add tests for this and mark it as completed as well.